pkg/controller/quay_io_ci_images_distributor/supplemental_images.go

```diff
@@ -0,0 +1,65 @@
+package quay_io_ci_images_distributor
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/util/gzip"
+)
+
+func LoadConfig(file string) (*SupplementalCIImagesConfig, error) {
```

Comment:
```suggestion
func LoadConfig(file string) (*Config, error) {
```

Reply:
I can't do that, beacsue `Config`  is already defined here: https://github.com/openshift/ci-tools/blob/master/pkg/controller/quay_io_ci_images_distributor/quay_io_ci_images_distributor.go#L392